docs/azure-saas-docs/content/en/_index.md

```diff
@@ -1,36 +1,57 @@
 ---
 type: docs
 no_list: false
-linkTitle: "Welcome"
+linkTitle: "Introduction"
 weight: 0
 ---
-# Welcome to the Azure SaaS Dev Kit
+# Welcome to the Azure SaaS Dev Kit!
 
 [Software as a Service (SaaS)](https://azure.microsoft.com/en-us/overview/what-is-saas/) doesn’t need to be complex and time consuming.
 
-The Azure SaaS Development Kit is a set of pre-built modular resources to help you launch your SaaS offering faster:
+The Azure SaaS Development Kit is a deployable reference implementation of pre-built resources designed to help you launch your SaaS offering faster:
 
-* Standard SaaS components deployable individually or mix-and-match
-* [Open-source code](https://github.com/Azure/azure-saas), allowing engineers to build by example or modify/extend to be purpose-built
-* Fully documented code allows for self-serve usage
+* It includes commonly needed SaaS components that implement features such as identity, onboarding, and tenant management.
+* It is 100% [Open-source code](https://github.com/Azure/azure-saas), allowing you to build by example or modify/extend to be purpose-built for your particular scenario.
+* The code is fully documented, making it clear how the code functions, and why key design decisions were made.
 
-Whether you're modernizing an existing application, building a new application, or migrating your application, the SaaS dev kit can help you.
+## Usage Options
 
-## Modules & Architecture
+How you use the dev kit is up to you, here are some ideas to get you started:
 
-![](futurestate.drawio.png)
+* Modernizing an existing application to support [full multitenancy](https://docs.microsoft.com/en-us/azure/architecture/guide/multitenant/considerations/tenancy-models#fully-multitenant-deployments) as part of a shift to a SaaS based business model.
+* Developing a greenfield SaaS offering for the first time.
+* Migrating a SaaS offering from another cloud to Azure.
 
-- [**B2C Authentication Service**](/services/b2c-auth-service/) - Provides a flexible identity solution.
-- [**Core App**](/services/core-app/)
-	- A web app where your customers to view plans and onboard to your solution.
-	- Provides you with tenant administration capabilities. (modify/remove/etc.)
-- [**SaaS.Application**](/services/saas-application/) - A sample application that you can extend or replace with your own code.
+This is not a one-size-fits-all solution. Use as little or as much as you like. It is designed to be both a modular deployable reference implementation and also a reference architecture. You are free to use and change the code contained within this project in any way you'd like (following the terms of the license):
 
-## Product SaaS Reference
+* Deploy the entire solution to Azure using the bicep templates we provide, make changes to fit your exact use case, and start building your SaaS application from there.
```

Comment:
Given that this project is intended for people to migrate into Azure for the first time, it would be good to link the Bicep documentation when mentioning it:
[Bicep](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/)